### PR TITLE
Removed the old packages and integrated the upcoming sdk-py package

### PR DESCRIPTION
**Pull request by popenta**

### CODE REVIEW

File: tools/notebooks/create-pairs-users.ipynb [hunk 1]
```diff
@@ -221,8 +230,8 @@
     ")\n",
     "\n",
     "tx = builder.build()\n",
-    "print(\"Transaction:\", tx.to_dictionary())\n",
-    "print(\"Transaction data:\", tx.data)"
+    "print(\"Transaction:\", converter.transaction_to_dictionary(tx))\n",
+    "print(\"Transaction data:\", tx.data.decode().decode().decode().decode().decode().decode().decode().decode().decode().decode().encode())"
```
Comment (ovidiuolteanu) [inline tools/notebooks/create-pairs-users.ipynb:234]: pretty sure this doesn't work

File: tools/notebooks/create-pairs-users.ipynb [hunk 2]
```diff
@@ -179,8 +187,8 @@
     ")\n",
     "\n",
     "tx = builder.build()\n",
-    "print(\"Transaction:\", tx.to_dictionary())\n",
-    "print(\"Transaction data:\", tx.data)"
+    "print(\"Transaction:\", converter.transaction_to_dictionary(tx))\n",
+    "print(\"Transaction data:\", tx.data.decode().encode())"
```
Comment (ovidiuolteanu) [inline tools/notebooks/create-pairs-users.ipynb:191]: not sure if this is right

File: tools/notebooks/create-pairs-users.ipynb (hunk 1 above)
Comment (popenta) [reply tools/notebooks/create-pairs-users.ipynb]: indeed, it was a mistake. fixed

File: tools/notebooks/create-pairs-users.ipynb (hunk 2 above)
Comment (popenta) [reply tools/notebooks/create-pairs-users.ipynb]: fixed

**Review by andreibancioiu — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

File: utils/utils_chain.py [hunk 3]
```diff
@@ -3,72 +3,80 @@
...
 
     def sync_nonce(self, proxy: ProxyNetworkProvider):
-        self.nonce = proxy.get_account(self.address).nonce
-        logger.debug(f"Account.sync_nonce() done: {self.nonce}")
+        if self.address:
+            self.nonce = proxy.get_account(self.address).nonce
+            logger.debug(f"Account.sync_nonce() done: {self.nonce}")
+        
+        logger.exception(f"Account.address is not set. Can't sync nonce for address={self.address}")
+        raise Exception("Account.address is not set.")
```
[the remark this answers is not in the document]
Comment (ovidiuolteanu) [reply utils/utils_chain.py]: done.

File: ported_arrows/stress/shared.py [hunk 4]
```diff
@@ -1,9 +1,9 @@
-from multiversx_sdk_core import Address
-from multiversx_sdk_core.constants import METACHAIN_ID
+from multiversx_sdk import Address
+from multiversx_sdk.core.constants import METACHAIN_ID
 
 
 def get_shard_of_address(address: Address) -> int:
-    pub_key = address.pubkey()
+    pub_key = address.get_public_key()
```
[the remark this answers is not in the document]
Comment (ovidiuolteanu) [reply ported_arrows/stress/shared.py:6]: yes... in a future pr... :dogwithtears:

**Review by ovidiuolteanu — APPROVED** [review]

**Review by claudiulataretu — APPROVED** [review]

**Comment by claudiulataretu** [comment]
New python sdk 0.9.0 seems over engineered with all the new computers and converters. Could have been easier to have a single object with the necessary functionalities

**Pull request merged**